src/shogun/util/factory.h

```diff
@@ -106,7 +106,11 @@ namespace shogun
 		return result;
 	}
 
-	/** Create embed string features from string char features
+	/** Create embedded string features from string char features.
+	 * The new features has the same alphabet as the original features. Data of
+	 * the new features is obtained by calling `obtain_from_char` with the given
```

Comment:
Much better. One final thing, could you make doxygen link the class method here?